src/renderer/components/ClusterPage/useCloudSync.js

```diff
@@ -46,5 +48,12 @@ export const useCloudSync = (cloudUrl) => {
     };
   }, [clouds, cloudUrl]);
 
-  return { isCloudFetching, cloudStatus };
+  const syncCloud = () => {
```

Comment:
Since this is in a hook, I believe it's legal to do this:
```suggestion
  const handleSyncCloud = useCallback(() => {...}, cloudUrl);
```

And then return `handleSyncCloud` as the handler to use.

Reply:
Good idea, done.